pkg/jx/cmd/create_project.go

```diff
@@ -0,0 +1,158 @@
+package cmd
+
+import (
+	"fmt"
+	"github.com/jenkins-x/jx/pkg/jx/cmd/templates"
+	"github.com/jenkins-x/jx/pkg/util"
+	"github.com/spf13/cobra"
+	"gopkg.in/AlecAivazis/survey.v1/terminal"
+	"io"
+	"os"
+)
+
+const (
+	createQuickstartName = "Create new application from a Quickstart"
+	createCamelName      = "Create new Apache Camel microservice"
+	createSpringName     = "Create new Spring Boot microservice"
+	importDirName        = "Import existing code from a directory"
+	importGitName        = "Import code from a git repository"
+	importGitHubName     = "Import code from a github repository"
+)
+
+var (
+	createProjectNames = []string{
+		createQuickstartName,
+		createCamelName,
+		createSpringName,
+		importDirName,
+		importGitName,
+		importGitHubName,
+	}
+
+	createProjectLong = templates.LongDesc(`
+		Creates a project
+`)
+
+	createProjectExample = templates.Examples(`
+		# Create a project
+		jx create project
+	`)
+)
+
+// CreateProjectWizardOptions the options for the command
+type CreateProjectWizardOptions struct {
+	CreateOptions
+}
+
+// NewCmdCreateProject creates a command object for the "create" command
+func NewCmdCreateProject(f Factory, in terminal.FileReader, out terminal.FileWriter, errOut io.Writer) *cobra.Command {
+	options := &CreateProjectWizardOptions{
+		CreateOptions: CreateOptions{
+			CommonOptions: CommonOptions{
+				Factory: f,
+				In:      in,
+				Out:     out,
+				Err:     errOut,
+			},
+		},
+	}
+
+	cmd := &cobra.Command{
```

Comment:
We should maybe retie the other commands for simplicity and always instruct the user to use the `create project` command instead of `create quickstart` or `import`.

Reply:
though I do like the specific commands too - it also allows much more powerful CLI options and completion

Reply:
the top level `jx create project` acts as a handy wizard to get folks going until they learn the lower level richer commands I guess?